test/mergeAccountData.spec.js

```diff
@@ -93,12 +97,16 @@ describe('mergeAccountData', function(){
           firstname: false,
           lastname: true,
           salutation: false,
+          '_Favorite Color': true,
+          '_Shoe Size': true,
         },
         personal_data: {
           email: 'email2',
           firstname: 'firstname2',
           lastname: 'lastname2',
           salutation: '',
+          '_Favorite Color': 'Purple',
```

Comment:
I'd add `_Favorite Color` to the `right` arg in this test to verify that if custom key that is in both `left` and `right` it uses the `right` value.

Reply:
what is a `left` and `right` value?

Reply:
`function mergeAccountData (left, right){`